src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/PrintReadsSparkIntegrationTest.java

```diff
@@ -220,4 +223,30 @@ public void testReadFiltering() throws IOException {
         runCommandLine(args.getArgsArray());
         SamAssertionUtils.assertSamsEqual(outBam, new File(getTestDataDir(), "expected.print_reads_one_malformed_read.bam"));
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testReadFiltering_asIntegrationTestSpec() throws IOException {
```

Comment:
Just replace the original version of the test with the (correct) `IntegrationTestSpec` version, and move the unit testing of `IntegrationTestSpec` itself into an `IntegrationTestSpecUnitTest`